sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -204,6 +204,61 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
     }
   }
 
+  test("Refresh Qualified Tables") {
+    withTempDatabase { db =>
+      withTempView("cachedTable") {
+        try {
+          sql(s"CREATE TABLE $db.cachedTable STORED AS PARQUET AS SELECT 1")
+          sql(s"CACHE TABLE $db.cachedTable")
+          assertCached(sql(
+            s"select * from $db.cachedTable"),
+            s"`$db`.`cachedTable`")
+          assert(spark.catalog.isCached(s"$db.cachedTable"),
+            s"Table '$db.cachedTable' should be cached")
+
+          sql(s"REFRESH TABLE $db.cachedTable")
+          assertCached(sql(s"select * from $db.cachedTable"), s"`$db`.`cachedTable`")
+          assert(spark.catalog.isCached(s"$db.cachedTable"),
+            s"Table '$db.cachedTable' should be cached after refresh")
+
+          activateDatabase(db) {
+            assertCached(sql("select * from cachedTable"), s"`$db`.`cachedTable`")
+            assert(spark.catalog.isCached("cachedTable"),
+              "Table 'cachedTable' should be cached after refresh")
+
+            sql(s"REFRESH TABLE cachedTable")
+            assertCached(sql("select * from cachedTable"), s"`$db`.`cachedTable`")
+            assert(spark.catalog.isCached("cachedTable"),
+              "Table 'cachedTable' should be cached after refresh")
+          }
+        } finally {
+          sql(s"UNCACHE TABLE $db.cachedTable")
+          sql(s"DROP TABLE $db.cachedTable")
+        }
+      }
+    }
+  }
+
+  test("Refresh Unqualified Tables") {
+    withTempView("cachedTable") {
+      try {
+        sql(s"CREATE TABLE cachedTable STORED AS PARQUET AS SELECT 1")
+        sql(s"CACHE TABLE cachedTable")
+        assertCached(sql(s"select * from cachedTable"), s"`cachedTable`")
```

Comment:
Nit: when there is no expression in string literal avoid string interpolation (`s"`)

Reply:
just removed the unnecessary string interpolation.